plugins/node/opentelemetry-plugin-pg/package.json

```diff
@@ -57,12 +57,13 @@
     "gts": "^1.1.0",
     "mocha": "^6.2.0",
     "nyc": "^15.0.0",
-    "pg": "^7.12.1",
+    "pg": "^8.0.3",
     "rimraf": "^3.0.0",
     "ts-mocha": "^6.0.0",
     "ts-node": "^8.6.2",
     "tslint-consistent-codestyle": "^1.16.0",
     "tslint-microsoft-contrib": "^6.2.0",
+    "cross-env": "^6.0.0",
```

Comment:
why do we need `cross-env` ? Because there are multiple versions ?

Reply:
This package is using it to run the tests locally and I saw it was missing. Are we relying on certain dev dependencies being hoisted via lerna?

The version does not matter to this package.

Reply:
It's being used on line 11. We probably have just gotten lucky that it is also in the dependency tree somewhere else and was installed. We do not use hoist.